nautobot/extras/forms/forms.py

```diff
@@ -1132,6 +1139,10 @@ class JobBulkEditForm(NautobotBulkEditForm):
         required=False,
         help_text="If checked, the values of has sensitive variables will be reverted to the default values defined in each Job's source code",
     )
+    is_singleton_override = forms.BooleanField(
+        required=False,
+        help_text="If checked, the values of has sensitive variables will be reverted to the default values defined in each Job's source code",
```

Comment:
```suggestion
        help_text="If checked, the values of is singleton will be reverted to the default values defined in each Job's source code",
```

There's got to be a better way to phrase these though :-)